metrics/prometheus.go

```diff
@@ -28,6 +31,7 @@ type prometheusRec struct {
 	bulkProcessed                  *prometheus.CounterVec
 	bulkTimeouts                   *prometheus.CounterVec
 	cbStateChanges                 *prometheus.CounterVec
+	cbCurrentCondition             *prometheus.GaugeVec
```

Comment:
Uniformity naming: `cbCurrentCondition` -> `cbCurrentState`

Reply:
Will make this change